README.md

```diff
@@ -1,16 +1,67 @@
-# Title
+# Iterator Drill: Find
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Define how the `filter()` method works
+* Demonstrate `filter`
 
 ## Introduction
 
-## SWBAT 1
+Filtering is a very common function of programming that we've all utilized at
+some point in our lives. When shopping for clothing online, chances are, we've
+filtered by sizes, so we can weed out the clothing that we definitely can't fit
+into, and browse through what caters to our needs. If you've ever ordered food
+online, chances are, we've all filtered by what's open right now, and what's
+available for delivery in your neighborhood--filtering out results for restaurants
+that don't deliver to your address. It is a very common action to need to perform.
+This used to be quite annoying to do manually, but now there is no need to fret--
+`filter` is here to help!
 
-## SWBAT 2
+## 
+
+You can think of `filter` as a for loop that specifically keeps or filters out
+certain values from an array. Consider the following code:
+
+```js
+let arr = [1, 2, 3, 4, 5, 6];
+let even = [];
+for(var i = 0; i < arr.length; i++) {
+  if (arr[i] % 2 === 0) even.push(arr[i]);
+}
+// even = [2,4,6]
+```
+
+This code checks all of the values in the array. If the value can be divided by 2, it
+is considered an even number. Those values are then pushed onto the even `array`. This
+works, but now we have the `filter` method to achieve the same result much easier.
+
+## Demonstrate `filter`
+
+Filter is similar to map. If you know `map`, you know `filter`! Filter receives the
+same arguments as map, and works very similarly. The only difference is that the
+callback needs to return either _true_ or _false_. If it returns _true_, the array
+keeps that element. If it returns false that element is filtered out.
```

Comment:
I think that we should provide a definition of `filter` that doesn't rely solely on `map`, or at least reiterate exactly what `map` is so that if someone doesn't know `map` or needs a refresher, they can catch up.